src/main/java/frc/robot/Robot.java

```diff
@@ -44,6 +51,26 @@
   private double maxPitchBackwardDegrees = -15; // placeholder value
   // assuming that pitch forward is positive and pitch backward is negative
   */
+
+  private final I2C.Port i2cPortDef = I2C.Port.kOnboard;
+  // private final I2C.Port i2cPortMXP = I2C.Port.kMXP;
+
+  private final ColorSensorV3 colorSensor1 = new ColorSensorV3(i2cPortDef);
```

Comment:
Nit: is it possible to give this color sensor a better variable name instead of `colorSensor1`? Maybe if we are planning to have multiple color sensors connected, then we can create an ArrayList and access each color sensor with an index like `colorSensors[1]`?

Any thoughts/opinions on this?

Reply:
Each colour sensor will probably have a separate purpose, so we could name it based on its use.

I think that would be less complex than creating an array of them.

Reply:
Sounds good to me! Let's just leave it as `colorSensor1` for now then.